Sources/SwiftSyntax/generated/SyntaxRewriter.swift

```diff
@@ -23,11 +23,11 @@
 
 open class SyntaxRewriter {
   public let viewMode: SyntaxTreeViewMode
-  
```

Comment:
Oh, a lot of these is just `swift-format` 508.0.1.

Reply:
https://github.com/apple/swift-syntax/blob/b4749403093de49b11418d7bd80f7ee2b4a926dd/CONTRIBUTING.md?plain=1#L26

`swift-format` doesn't format generated code, so I think something went awry during the code generation process. Would you mind re-running the code generation to try to fix it?